CONTRIBUTING.md

```diff
@@ -15,35 +24,42 @@ Did you find a bug in GeoBlacklight or interested in a new feature? Make sure to
     - Explain the actual behavior
     - Provide steps to reproduce the actual behavior
 
-### GeoBlacklight Software Versioning
-GeoBlacklight follows the practice of [Semantic Versioning](https://semver.org/) for software releases. The declared semantically versioned public API includes:
- - the GeoBlacklight-Schema
- - the [public GeoBlacklight Ruby codebase classes](http://www.rubydoc.info/github/geoblacklight/geoblacklight/master/frames)
- - the GeoBlacklight JavaScript interface
- - the GeoBlacklight view interface
+### Issue Assignment
+Issues are used in the GeoBlacklight community as a convenient way to document bugs, propose new functionality and enhancements, and discuss important code changes.
+
+- If someone is already assigned to an issue, that does not mean you can't contribute.
+
 
 ### Contributing code or documentation
 GeoBlacklight welcomes code and documentation contributions. You do *not* need to be a "GeoBlacklight Committer" to contribute code or documentation. We follow the [pull request](https://help.github.com/articles/using-pull-requests/) model for contributing on GitHub. GeoBlacklight uses a suite of tests to express its features and protect from bugs:bug:.
```

Comment:
Suggest making this a level 2 (##) as well. Also, the `:bug:` is not displaying properly - I think adding a space before the first colon might work?